docs/content/docs/deployment/azure-frs.md

```diff
@@ -1,5 +1,94 @@
+
+```typescript
+const config: FrsConnectionConfig = {
+    tenantId: "myFrsTenantId",
+    // IMPORTANT: this token provider is suitable for testing ONLY. It is NOT secure.
+    tokenProvider: new InsecureTokenProvider("myFrsTenantKey", { id: "UserId", name: "Test User" }),
+    orderer: "https://myFrsOrdererUrl",
+    storage: "https://myFrsStorageUrl",
+}
+
+const client = new FrsClient(config);
+```
+
+Now that we have an instance of our client, we can start using it to create new or load existing Fluid containers!
+
+# Managing Containers
+
+The `FrsClient` provides two functions on its API to create and get containers respectively. They both take in two parameters:
+- A container config that defines the ID of the container and an optional entrypoint for logging
+- A container schema that defines the DDSes and data objects that this container will hold
+
+```typescript
+const schema = {
+    name: "my-container",
+    initialObjects: {
+        /* ... */
+    },
+    dynamicObjectTypes: [ /*...*/ ],
+}
+const frsClient = new FrsClient(config);
+await frsClient.createContainer({ id: "_unique-id_" }, schema);
+const { fluidContainer, containerServices } = await frsClient.getContainer({ id: "_unique-id_" }, schema);
+```
+
+The ID that is being passed in to the container config can be thought of as the "filename" for the item holding the data for this container on the FRS backend. Any client that wants to join the same collaborative session just needs to call `getContainer` with the same container ID.
+
+For the further information on how to start recording logs being emitted by Fluid, please see [Telemetry](../testing/telemetry.md)
+
+The container being fetched back will hold the `initialObjects` as defined in the container schema. Please read [Data modeling](../build/data-modeling.md) to see how to establish the schema and use the `fluidContainer` object.
+
+# Getting Audience Details
+
+Both calls to `createContainer` and `getContainer` return an `FrsResources` object that holds the Fluid container that we were discussing above as well as a `containerServices` object. Whereas the container itself will always stay the same regardless of which service it is being connected, the `containerServices` hold values that are specific to the FRS service. Within this object, we will find an `audience` value that can be used to manage the roster of users that are currently collaborating in the container.
```

Comment:
Is this deliberately not a tuple?

Reply:
It was originally a tuple but was changed to an object. I'm still not convinced that an object is the right pattern here and I really dislike that we have to define the return object type. @ChumpChief / @sumedhb1995 who drove this change.

Reply:
There were multiple requests from consumers of the create/getContainer call who found that indexing the returned value like `const fluidContainer = (await client.createContainer(....))[0]` when they only need one of the two values was affecting code readability and keeping it as a tuple wasn't actually restricting people's ability to add on additional returned values in the array.

Reply:
But wouldn't the idiomatic code look like `const [fluidContainer] = (await client.createContainer(....))`? To be clear the object is OK with me I just want to better understand the reasoning.

Reply:
Correct, but I believe the theory is that the developer could also technically write `const [containerServices, fluidContainer] = (await client.createContainer(....))` and they wouldn't see any error, even though they've swapped their returned values around. Or they could do `const fluidContainer = (await client.createContainer(....))[1]` and again get the wrong value. It relies on the developer and how they choose to parse the two values out. Whereas with the object, they need to explicitly get the `fluidContainer` or `containerServices` values from it